src/fs/fs.go

```diff
@@ -37,8 +38,8 @@ import (
 
 // DANGER DANGER DANGER
 // compile-time only options in case we need to disable checking the certs or https
-const DisableCertChecking = false
-const DisableHttps = false
+const DisableCertChecking = true
+const DisableHttps = true
```

Comment:
These two options should not be changed, please remove this change from the commit.

Reply:
@csoni111 Resolved this.